internal/sdkprovider/service/kafkaschema/kafka_schema.go

```diff
@@ -16,6 +17,29 @@ import (
 	"github.com/aiven/terraform-provider-aiven/internal/schemautil/userconfig"
 )
 
+// protobufHeaderRegExp is a regular expression that matches a valid Protobuf header.
+var protobufHeaderRegExp = regexp.MustCompile("syntax(.*|)=(.*|)proto3(.*|);")
```

Comment:
I think we will be better off with something like https://github.com/bufbuild/protovalidate-go or https://pkg.go.dev/github.com/deelawn/protoc-gen-validation. Since the protobuf format may evolve in the future, Aiven, in the long run, will also support various versions simultaneously. We want to delegate validation to dedicated projects that focus solely on that.

Reply:
currently it was implemented the same way as it is on front end, of course in the long run it will be viable to replace it

Reply:
I don't see any reason having such a validation. It basically checks that string contains `syntax = proto3;` substring. Which is not a valid syntax btw.
And I might be missing something `(.*|)` means "anything-or-nothing or nothing", right? The `plan` command won't show any "real" validation issues.